tests/framework/installer/ceph_installer.go

```diff
@@ -561,3 +567,111 @@ func NewCephInstaller(t func() *testing.T, clientset *kubernetes.Clientset, useH
+// WipeClusterDisks runs a disk wipe job on all nodes in the k8s cluster.
+func (h *CephInstaller) WipeClusterDisks(namespace string) error {
+	wipeJobName := func(node string) string {
+		return "rook-ceph-disk-wipe-" + node
+	}
+
+	// Wipe clean disks on all nodes
+	nodes, err := h.GetNodeHostnames()
+	if err != nil {
+		return fmt.Errorf("failed to get node hostnames. %+v", err)
+	}
+	for _, node := range nodes {
+		job := h.GetDiskWipeJob(node, wipeJobName(node), namespace)
+		_, err := h.k8shelper.KubectlWithStdin(job, createFromStdinArgs...)
+		if err != nil {
+			return fmt.Errorf("failed to create disk wipe job for host %s. %+v", node, err)
+		}
+	}
+
+	allJobsAreComplete := func() (done bool, err error) {
+		for _, node := range nodes {
+			j, err := h.k8shelper.Clientset.BatchV1().Jobs(namespace).Get(wipeJobName(node), metav1.GetOptions{})
+			if err != nil {
+				return false, nil
+			}
+			if j.Status.Failed > 0 {
+				return false, fmt.Errorf("job %s failed", wipeJobName(node))
+			}
+			if j.Status.Succeeded == 0 {
+				return false, nil
+			}
+		}
+		return true, nil
+	}
+
+	if err = wait.Poll(5*time.Second, 60*time.Second, allJobsAreComplete); err != nil {
+		return fmt.Errorf("failed to wait for wipe jobs to complete. %+v", err)
+	}
+
+	for _, node := range nodes {
+		// if delete fails, don't worry about the error; delete only on best-effort basis
+		h.k8shelper.Clientset.BatchV1().Jobs(namespace).Delete(wipeJobName(node), &metav1.DeleteOptions{})
+	}
+
+	return nil
+}
+
+// GetDiskWipeJob returns a YAML manifest string for a job which will wipe clean the extra
+// (non-boot) disks on a node, allowing Ceph to use the disk(s) during its testing.
+func (h *CephInstaller) GetDiskWipeJob(nodeName, jobName, namespace string) string {
+	// put the wipe job in the cluster namespace so that logs get picked up in failure conditions
+	return `apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ` + jobName + `
+  namespace: ` + namespace + `
+spec:
+    template:
+      spec:
+          restartPolicy: Never
+          containers:
+              - name: disk-wipe
+                image: ` + h.CephVersion.Image + `
+                securityContext:
+                    privileged: true
+                volumeMounts:
+                    - name: dev
+                      mountPath: /dev
+                command:
+                    - "sh"
+                    - "-c"
+                    - |
+                      set -xEeuo pipefail
+                      # some devices might still be mapped that lock the disks
+                      if dmsetup status | grep ceph; then
+                        dmstatus="$(dmsetup status | grep ceph)"
+                        for statusline in $dmstatus; do
+                          device_with_colon="${statusline%%\ *}" # the first 'word' is the device with a colon after it
+                          device="${device_with_colon%%:}" # remove the colon
+                          dmsetup remove "${device}"
```

Comment:
Is this enough to remove lvm metadata? 🤔

Reply:
I'm not certain. In my experience, the host doesn't have LVM data on it somehow. I tried the various LVM scan and list options, and nothing showed up for me recently. In the past that hasn't been the case, so I've been trying to see if I can get any to show up.

Reply:
Aha! I did keep getting strange issues about LVM stuff still being present when ceph-volume looked. It turns out that `vgimport -a` will start to show LVM stuff on the disks.